ui/src/app/routing/app-routing.tsx

```diff
@@ -77,11 +78,12 @@ const bindClients = () => {
   bindApiClients(
     new Configuration({
       basePath: getApiBaseUrl(),
-      accessToken: () => getAccessToken(),
+      // TODO: Look into removing 'Bearer' when updating to OpenAPI 3: https://stackoverflow.com/a/45471010/2166475
+      accessToken: () => 'Bearer ' + getAccessToken(),
```

Comment:
This could be an OpenApi change, but it appears like'Bearer' does not get appended to our access token in OpenApi 2. This may change in OpenApi 3. 

See https://stackoverflow.com/a/45471010/2166475 for details

Does 'Bearer' need to be added to notebooks and tanagra?